controller/src/main/java/io/pravega/controller/store/stream/ZKGarbageCollector.java

```diff
@@ -159,7 +159,7 @@ int getLatestBatch() {
 
     @VisibleForTesting
     CompletableFuture<Void> fetchVersion() {
-        return zkStoreHelper.getData(guardPath)
+        return zkStoreHelper.getData(guardPath, x -> x)
```

Comment:
Out of curiosity, wouldn't we be able to reduce this lambda to a single call to `getData` by replacing the `x -> x` argument with the function that deserializes it in line 163?

Reply:
we dont want any deserialization in this case hence whatever byte array we get, we do a no op. hence the lambda x->x